site/content/en/contact/_index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Contact Us"
+subtitle: ""
+description: "Is the DevOps Stack the right tool for you? Contact us to find out!"
+draft: false
+---
+
+### Let's talk
+
+Is the DevOps Stack the right tool for you? Contact us to find out!
+
+* **Phone: +41 21 619 10 10**
```

Comment:
I'm not sure if someone call this number the person that will answer the phone will be able to forward the call to us...